sup3r/postprocessing/collection.py

```diff
@@ -170,6 +173,19 @@ def get_slices(final_time_index, final_meta, new_time_index, new_meta):
 
         return row_slice, col_loc
 
+    @classmethod
+    def get_coordinate_indices(cls, target_meta, full_meta):
+        """Get coordindate indices in meta data for given targets"""
+        ll2 = np.vstack((full_meta.latitude.values,
+                         full_meta.longitude.values)).T
+        tree = KDTree(ll2)
+
+        targets = np.vstack((target_meta.latitude.values,
+                             target_meta.longitude.values)).T
+        _, indices = tree.query(targets, distance_upper_bound=0.001)
+        indices = [i for i in indices if i < len(full_meta)]
```

Comment:
Just curious, whats the reason for this filtering on max value? The tree query shouldn't return anything greater than the full meta?

Reply:
When I was testing this the query returned an index equal to the length of the meta if the query wasn't found. Maybe it only does this when you query a single point?

Reply:
oh yeah "Missing neighbors are indicated with self.n", i thought it used to be negative but i guess they updated it. Honestly i usually just do a mask with the returned distance and set the bad values to NaN or something else more explicit haha.

Reply:
oh thats a good idea